Formula/ghostscript.rb

```diff
@@ -1,17 +1,15 @@
 class Ghostscript < Formula
   desc "Interpreter for PostScript and PDF"
   homepage "https://www.ghostscript.com/"
-  url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9561/ghostpdl-9.56.1.tar.xz"
-  sha256 "05e64c19853e475290fd608a415289dc21892c4d08ee9086138284b6addcb299"
+  url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs1000/ghostpdl-10.0.0.tar.xz"
+  sha256 "8f2b7941f60df694b4f5c029b739007f7c4e0d43858471ae481e319a967d5d8b"
   license "AGPL-3.0-or-later"
-  revision 1
 
   # We check the tags from the `head` repository because the GitHub tags are
   # formatted ambiguously, like `gs9533` (corresponding to version 9.53.3).
   livecheck do
-    url :stable
-    regex(/href=.*?ghostpdl[._-]v?(\d+(?:\.\d+)+)\.t/i)
-    strategy :github_latest
+    url :head
+    regex(/^ghostpdl[._-]v?(\d+(?:\.\d+)+)$/i)
```

Comment:
Livecheck was failing, which I think was due to number of uploaded files. They don't show up on detected release page. They do show up on https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/expanded_assets/gs1000

Went and reverted livecheck to previous one as mentioned in comment for livecheck.

Reply:
This failure is due to the changes described in Homebrew/brew#13853 (i.e., GitHub doesn't include the asset list in the HTML now). I was about to create a PR to update this `livecheck` block but I'll add some suggestions here instead.